client.go

```diff
@@ -200,6 +201,42 @@ func (c *Client) GetPNCounter(ctx context.Context, name string) (*PNCounter, err
 	return c.proxyManager.getPNCounter(ctx, name)
 }
 
+// GetDistributedObjectsInfo returns the information of all objects created cluster-wide.
+func (c *Client) GetDistributedObjectsInfo(ctx context.Context) ([]types.DistributedObjectInfo, error) {
+	if atomic.LoadInt32(&c.state) != ready {
+		return nil, hzerrors.ErrClientNotActive
+	}
+
+	request := codec.EncodeClientGetDistributedObjectsRequest()
+	resp, err := c.proxyManager.invokeOnRandomTarget(ctx, request, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	objects := codec.DecodeClientGetDistributedObjectsResponse(resp)
+	result := make([]types.DistributedObjectInfo, 0, len(objects))
+
+	cached := make(map[types.DistributedObjectInfo]struct{})
+	for _, o := range c.proxyManager.getCachedObjectsInfo() {
+		cached[o] = struct{}{}
+	}
+
+	for _, o := range objects {
+		if !serviceSupported(o.ServiceName()) {
```

Comment:
There's no need to check whether the service is supported, let's return everything.

Reply:
addressed in f2a4add